Mathlib/LinearAlgebra/Matrix/Charpoly/Coeff.lean

```diff
@@ -162,6 +162,11 @@ theorem eval_det (M : Matrix n n R[X]) (r : R) :
   symm
   exact matPolyEquiv_eval _ _ _ _
 
+theorem det_one_sub_mul_comm (M : Matrix m n R) (N : Matrix n m R) :
```

Comment:
Or maybe the better thing would be to make LinearAlgebra.Matrix.SchurComplement not import LinearAlgebra.Matrix.PosDef, and then LinearAlgebra.Matrix.SchurComplement can be imported here?